roman_numerals/lib/roman_numerals.ex

```diff
@@ -1,6 +1,22 @@
 defmodule ROMAN_NUMERALS do
   def convert(0), do: ""
-  def convert(n) when n>=1 do
+  def convert(n) when n<=3 do
    "I" <> convert(n-1)
   end
+
+  def convert(n) when n>=10 do
+    "X" <> convert(n-10)
+  end
+
+  def convert(n) when n>=9 do
+    "IX" <> convert(n-9)
+  end
+
+  def convert(n) when n>=5 do
+    "V" <> convert(n-5)
+  end
+
+  def convert(n) when n>=4 do
+    "IV" <> convert(n-4)
+  end
```

Comment:
You seem to have a good pattern here. Is there a way we can make this more generic?